README.md

```diff
@@ -1,30 +1,129 @@
 # HardwareCheckout
+
 A simple website which will facilitate physical hardware checkout
 
-# Server
+## Server Installation
+
+Step 1 - git clone
+
+```
+git clone https://github.com/5hu5ky/HardwareCheckout.git
```

Comment:
This should probably point to `https://github.com/linted/HardwareCheckout`. Since this is a pull request it doesn't make much sense to advertise cloning a different repo.

Reply:
done